### PR TITLE
Doc changes for new API enhancement.

### PR DESCRIPTION
**Pull request by tabhus**

### CODE REVIEW

File: report-results/changelog/index.md [hunk 1]
```diff
@@ -4,6 +4,16 @@ title: What's New
 
 # GIA Report Results API Changelog
 
+## 2021-10-31
+
+#### Changed
+
+- The query [`getReport`](https://gialaboratory.github.io/report-results/reference/query.doc.html) now also returns an array of new object [`assets`](https://gialaboratory.github.io/report-results/reference/reportasset.doc.html) to provide list of assets linked to a reoprt.
```
Comment (mattgrogan) [inline report-results/changelog/index.md:11]: Please fix spelling of "report"

File: report-results/example-queries/index.md [hunk 2]
```diff
@@ -209,6 +209,69 @@ result
 }
 ```
 
+## Query Report Containing Multiple Linked Assets
```
Comment (mattgrogan) [inline report-results/example-queries/index.md:212]: Please add the new fields to the master query (instead of creating a new sample query)

Comment (mattgrogan) [reply report-results/example-queries/index.md:212]: Now I see maybe you did this already, if so, let's delete this "extra" query. Let me know if I am misunderstanding something.

Comment (mattgrogan) [reply report-results/example-queries/index.md:212]: @tabhus you want to leave this in?

**Comment by tabhus** [comment]
Yes.

**Pull request merged**